applications/nrf5340_audio/src/bluetooth/Kconfig

```diff
@@ -354,6 +354,12 @@ config BT_AUDIO_BROADCAST_ID_FIXED
 	  Fixed broadcast ID; 3 octets. Will only be used if BT_AUDIO_USE_BROADCAST_ID_RANDOM=n.
 	  Should only be used for debugging.
 
+config BT_AUDIO_INTERLEAVED
```

Comment:
Pure curiosity: is there a reason to omit "packing" from the name of the Kconfig?

Reply:
No, not really. Just don't like long CONFIGs, but I can add it